reference/api/searchable_attributes.md

```diff
@@ -42,6 +42,10 @@ List the settings.
 
 Update the [searchable attributes](/reference/features/field_properties.md#searchable-fields) of an index.
 
+::: warning
+Manually updating `searchableAttributes` will change the order of the returned documents' fields.
```

Comment:
```suggestion
Manually updating `searchableAttributes` will change the displayed order of the returned documents' fields in the JSON response.
```

Reply:
Maybe my sentence is too long, I just don't want the users to think it will change another kind of order, like the attribute order for the relevancy

Reply:
This is great, just a minor alteration
```suggestion
Manually updating `searchableAttributes` will change the displayed order of document fields in the JSON response.
```